ui/src/app/machines/views/MachineDetails/MachineConfiguration/PowerForm/PowerParameters/PowerParameterDefinition/PowerParameterDefinition.test.tsx

```diff
@@ -0,0 +1,56 @@
+import { render, screen } from "@testing-library/react";
+
+import PowerParameterDefinition from "./PowerParameterDefinition";
+
+import { PowerFieldType } from "app/store/general/types";
+import { powerField as powerFieldFactory } from "testing/factories";
+
+it("renders the value of a power parameter", () => {
+  const field = powerFieldFactory({
+    field_type: PowerFieldType.STRING,
+  });
+  render(<PowerParameterDefinition field={field} powerParameter="parameter" />);
+
+  expect(screen.getByText(/parameter/)).toBeInTheDocument();
+});
+
+it("handles 'choice' power fields", () => {
+  const field = powerFieldFactory({
+    choices: [
+      ["choice1", "Choice 1"],
+      ["choice2", "Choice 2"],
+    ],
+    field_type: PowerFieldType.CHOICE,
+  });
+  render(<PowerParameterDefinition field={field} powerParameter="choice1" />);
+
+  expect(screen.getByText(/Choice 1/)).toBeInTheDocument();
+});
+
+it("handles 'multiple_choice' power fields", () => {
+  const field = powerFieldFactory({
+    choices: [
+      ["choice1", "Choice 1"],
+      ["choice2", "Choice 2"],
+      ["choice3", "Choice 3"],
+    ],
+    field_type: PowerFieldType.MULTIPLE_CHOICE,
+  });
+  render(
+    <PowerParameterDefinition
+      field={field}
+      powerParameter={["choice1", "choice2"]}
+    />
+  );
+
+  expect(screen.getByText(/Choice 1, Choice 2/)).toBeInTheDocument();
```

Comment:
I wonder whether we should use a [Definition Term](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) element in the underlying [Definition](https://github.com/canonical-web-and-design/maas-ui/blob/master/ui/src/app/base/components/Definition/Definition.tsx#L21) component.
I decided against it when I was working on the Definition component as a Definition Term has to be a part of a [Description List](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) and it felt like an overkill in a lot of cases where we use it for just a single item.

But in this case it would make for better semantics (and a11y). It looks like a classic example for when Definition List could be used? I wonder what are your thoughts on this.

> Description lists are useful for displaying metadata as a list of key-value pairs.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl#metadata

Reply:
Yeah that sounds like a good idea. Maybe we could upstream this to Vanilla itself? I feel like a component that displays key-value pairs (where the value could be a list) should be standardised. Regardless I've added an issue (#3751) to track it.

Reply:
Awesome, thanks @Caleb-Ellis